diff --git a/node_modules/react-native-record-screen/ios/RecordScreen.mm b/node_modules/react-native-record-screen/ios/RecordScreen.mm
index 67ac51f..a23778e 100644
--- a/node_modules/react-native-record-screen/ios/RecordScreen.mm
+++ b/node_modules/react-native-record-screen/ios/RecordScreen.mm
@@ -290,12 +290,12 @@ RCT_REMAP_METHOD(clean,
 }
 
 // Don't compile this code when we build for the old architecture.
-#ifdef RCT_NEW_ARCH_ENABLED
-- (std::shared_ptr<facebook::react::TurboModule>)getTurboModule:
-    (const facebook::react::ObjCTurboModule::InitParams &)params
-{
-    return std::make_shared<facebook::react::NativeRecordScreenSpecJSI>(params);
-}
-#endif
+// #ifdef RCT_NEW_ARCH_ENABLED
+// - (std::shared_ptr<facebook::react::TurboModule>)getTurboModule:
+//     (const facebook::react::ObjCTurboModule::InitParams &)params
+// {
+//     return std::make_shared<facebook::react::NativeRecordScreenSpecJSI>(params);
+// }
+// #endif
 
 @end
